Admin Installation Steps (#368)
mattermost/mattermost-plugin-incident-management: docs/administrator-guide/installation-and-activation.md

## Installing Mattermost Incident Management for Self-Managed Deployments

Mattermost Incident Management is available in the Plugin Marketplace. You can download and install the plugin from Mattermost.

1. Open **System Console > Plugin Management**.
2. Search for **Incident Response** using the search bar or scroll through the list manually.
3. Select **Install**.
4. Next, select **Configure**.
5. Select **true** to enable the plugin.
6. Select **Save**.

When you open the Main Menu, **Playbooks & Incidents** is available as a menu item.

## Activating Mattermost Incident Management for Cloud Deployments

> Mattermost Incident Management is included in the Mattermost Cloud workspace but needs to be activated prior to use.

Reviewer: Every Cloud account comes with this feature enabled by default.

> Mattermost Incident Management is included in the Mattermost Cloud workspace but needs to be activated prior to use.

Author: ```suggestion
Mattermost Incident Management is included in the Mattermost Cloud workspace and is enabled by default.
```
